packages/riipen-ui-docs/src/modules/components/MarkdownPage.jsx

```diff
@@ -11,141 +18,164 @@ import Menu from "src/modules/components/Menu";
+              }
+
+              const name = options.demo;
+
+              if (!demos || !demos[name]) {
+                throw new Error(
+                  [
+                    `Missing demo: ${name}. You can use one of the following:`,
+                    Object.keys(demos)
+                  ].join("\n")
                 );
               }
 
-              return <MarkdownElement key={index} text={section} />;
-            })}
-            <div className="footer">
-              <Divider />
-              <Footer />
-            </div>
-          </div>
-          <div className="toc">
-            <TOC sections={sections} />
+              return <Demo key={index} demo={demos[name]} options={options} />;
+            }
+
+            return <MarkdownElement key={index} text={section} />;
+          })}
+          <div className="footer">
+            <Divider />
+            <Footer />
           </div>
         </div>
-        <style jsx>{`
-          .page {
-            display: flex;
-            justify-content: space-between;
-            padding-top: 22px;
-            position: relative;
-            top: 50px;
-            width: 100%;
-          }
-          .page :global(*:target) {
-            padding-top: 50px;
-          }
-          .content {
-            padding: 0 20px;
-            overflow-x: auto;
-            width: 100%;
-          }
-          .menu {
-            padding: ${theme.spacing(2)}px;
-          }
+        <div className="toc">
+          <TOC sections={sections} />
+        </div>
+      </div>
+      <style jsx>{`
+        .page {
+          display: flex;
+          justify-content: space-between;
+          padding-top: 22px;
+          position: relative;
+          top: 50px;
+          width: 100%;
+        }
+        .page :global(*:target) {
+          padding-top: 50px;
+        }
+        .content {
+          padding: 0 20px;
+          overflow-x: auto;
+          width: 100%;
+        }
+        .menu {
+          padding: ${theme.spacing(2)}px;
+        }
+        .menuMobile {
+          display: none;
+        }
+        .menuMobile :global(.popover) {
+          overflow-y: auto;
+        }
+        .toc {
+          top: 70px;
+          position: sticky;
+          max-width: 250px;
+          min-width: 175px;
+        }
+        @media (max-width: ${theme.breakpoints.md}px) {
+          .menu,
           .toc {
-            top: 70px;
-            position: sticky;
-            max-width: 250px;
-            min-width: 175px;
-          }
-          @media (max-width: ${theme.breakpoints.md}px) {
-            .menu,
-            .toc {
-              display: none;
-            }
+            display: none;
           }
-          .footer {
-            margin: ${theme.spacing(10)}px 0;
-          }
-          .footer :global(footer) {
-            margin-top: ${theme.spacing(10)}px;
+
+          .menuMobile {
+            display: initial;
+            margin-left: auto;
```

Comment:
More hackery to make the menu bars icon display on the far right of the app bar (just passing in classes prop of AppBar wasn't working

Reply:
We should fix `classes` to app bar if they aren't woring instead of this. I imagine you need to pass a CSS class to Appbar that is something like:

```css
.appbar {
  display: flex;
  justify-content: space-between;
}
```

Reply:
Figured it out, need :global(.classToPassToAppBar) in MarkdownPage!